larq_compute_engine/mlir/python/end2end_test.py

```diff
@@ -0,0 +1,79 @@
+import sys
+import pytest
+import larq as lq
+import subprocess
+import tensorflow as tf
+import numpy as np
+from tensorflow.python.eager import context
+
+from larq_compute_engine.mlir.python.converter import convert_keras_model
+
+
+@pytest.fixture
+def eager_mode():
+    """pytest fixture for running test in eager mode"""
+    with context.eager_mode():
+        yield
```

Comment:
I don't think we need this here since the test will always be run with TensorFlow 2.x which has eager execution enabled by default.
```suggestion
```